elfpy/utils/apeworx_integrations.py

```diff
@@ -27,26 +28,62 @@
     from ethpm_types.abi import MethodABI
 
 
-class HyperdriveProject(ProjectManager):
-    """Hyperdrive project class, to provide static typing for the Hyperdrive contract."""
+def to_fixed_point(float_var, decimal_places=18):
+    """Convert floating point argument to fixed point with specified number of decimals."""
+    return int(float_var * 10**decimal_places)
 
-    hyperdrive: ContractContainer
-    address: str = "0xB311B825171AF5A60d69aAD590B857B1E5ed23a2"
 
-    def __init__(self, path: Path) -> None:
-        """Initialize the project, loading the Hyperdrive contract."""
-        if path.name == "examples":  # if in examples folder, move up a level
-            path = path.parent
-        super().__init__(path)
-        self.load_contracts()
-        try:
-            self.hyperdrive: ContractContainer = self.get_contract("Hyperdrive")
-        except AttributeError as err:
-            raise AttributeError("Hyperdrive contract not found") from err
+def to_floating_point(float_var, decimal_places=18):
+    """Convert fixed point argument to floating point with specified number of decimals."""
+    return float(float_var / 10**decimal_places)
+
 
-    def get_hyperdrive_contract(self) -> ContractInstance:
-        """Get the Hyperdrive contract instance."""
-        return self.hyperdrive.at(self.conversion_manager.convert(self.address, AddressType))
+def get_market_state_from_contract(contract: ContractInstance, **kwargs) -> hyperdrive_market.MarketState:
+    """Return the current market state from the smart contract.
```

Comment:
Add some newlines before Parameters & Returns to make sure formatting in autodocs is correct